src/main/java/entity/ChatChannel.java

```diff
@@ -0,0 +1,34 @@
+package entity;
+
+public class ChatChannel {
+
+
+
+    private String channelURL;
+    private String user1Id;
+    private String user2Id;
+    private String lastMessage;
+
+    public ChatChannel(String channelURL, String lastMessage, String user1Id, String user2Id) {
+        this.channelURL = channelURL;
+        this.lastMessage = lastMessage;
+        this.user1Id = user1Id;
+        this.user2Id = user2Id;
+    }
+
+    public String getChannelURL() {
+        return channelURL;
+    }
+
+    public String getLastMessage() {
```

Comment:
Style: Perhaps we could write the getters in 1 line to mitigate overwhelming scrolls.
Double E's from me to you: Excellent Effort!